README.md

```diff
@@ -1,14 +1,26 @@
 # GatorGrouper
 
+![logo](docs/gatorgrouper_logo.svg "GatorGrouper")
```

Comment:
Shouldn't this be `![logo](.github//gatorgrouper_logo.svg** "GatorGrouper")`?

Reply:
Thanks for finding that. I just fixed it.